src/app/pages/loading-waiting-states/progress-bar-indeterminate/progress-bar-indeterminate.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ProgressBarIndeterminateComponent } from './progress-bar-indeterminate.component';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+import { MatProgressBarModule } from '@angular/material/progress-bar';
+import { MatSlideToggleModule } from '@angular/material/slide-toggle';
+import { SpacerModule } from '@pxblue/angular-components';
+import { EmptyStateModule } from '@pxblue/angular-components';
```

Comment:
You have duplicate import statements for angular-components. 

```suggestion
import { SpacerModule, EmptyStateModule } from '@pxblue/angular-components';
```